yarn.lock

```diff
@@ -2921,7 +2921,7 @@
     "@hapi/hoek" "8.x.x"
 
 "@ibm-watson/discovery-react-components@file:packages/discovery-react-components":
-  version "1.2.0-beta.10"
+  version "1.2.0-beta.12"
```

Comment:
Why this change?

Reply:
Happened when I tried to `yarn add cross-env`, not realizing that we already had it installed. I can undo that if we want, but it didn't seem to hurt anything, so I just left it in.

Reply:
Just realized it has `file:`. So this must have been added when linking to local source of the components. I assume this won't have any impact in production, so fine as is.